docs/intro/tutorial03.txt

```diff
@@ -449,6 +449,7 @@ file, go ahead and add an ``app_name`` to set the application namespace:
     :filename: polls/urls.py
 
     from django.conf.urls import url
+    from . import views
```

Comment:
ok and thank you very much

On Sat, Jan 2, 2016 at 3:43 PM, Tim Graham <notifications@github.com> wrote:

> In docs/intro/tutorial03.txt
> <https://github.com/django/django/pull/5914#discussion_r48683878>:
>
> > @@ -449,6 +449,7 @@ file, go ahead and add an ``app_name`` to set the application namespace:
> >      :filename: polls/urls.py
> >
> >      from django.conf.urls import url
> > +    from . import views
>
> It should look like the previous step:
>
> from django.conf.urls import url
>
> from . import views
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/django/django/pull/5914/files#r48683878>.
>



-- 
Regards,
Mahendra Yadav